chapters/en/chapter1/3.mdx

```diff
@@ -326,4 +352,60 @@ Like with text generation and summarization, you can specify a `max_length` or a
 
 </Tip>
 
-The pipelines shown so far are mostly for demonstrative purposes. They were programmed for specific tasks and cannot perform variations of them. In the next chapter, you'll learn what's inside a `pipeline()` function and how to customize its behavior.
+## Image and audio pipelines
+
+Beyond text, Transformer models can also work with images and audio. Here are a few examples:
+
+### Image classification
+
+```python
+from transformers import pipeline
+
+image_classifier = pipeline("image-classification")
```

Comment:
for the sake of consistency, let's put a `model_name` so that we have consistent behaviour.